app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
-    # Write your code here
-    pass
+
+    km = 0
+
+    def __init__(self, kilometer):
+        self.km = kilometer
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        return Distance(kilometer=self.km + other.km) \
+            if isinstance(other, Distance) \
+            else Distance(kilometer=self.km + other)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other):
+        return self.km < other.km \
+            if isinstance(other, Distance) else self.km < other
```

Comment:
Better not use ternary operator here: it’s good for example for returning one of two strings not for calculations and comparison. Fix this issue throughout your code